ml-package-versions.yml

```diff
@@ -0,0 +1,100 @@
+sklearn:
+  package_info:
+    pip_release: "scikit-learn"
+    pip_dev: |
+      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
+
+  models:
+    minimum: "0.23.2"
```

Comment:
Temporarily use this value to avoid creating too many github actions jobs.

Reply:
@dbczumar btw how do we determine the minimum version to support?

Reply:
@harupy Good question! For scikit-learn autologging, that's fortunately well-defined. For other integrations, I'd recommend that we go back and take a look at when we introduced them. For integrations that are >= 6 months old, it seems reasonable to use the library minor version at the time of introduction. For newer integrations (i.e. PyTorch lightning, perhaps @smurching has input)